packages/charts/src/charts/table/table-chart.component.tsx

```diff
@@ -97,6 +105,9 @@ export const TableChart = ({
     [maxScroll, overflowLeft, overflowRight]
   );
 
+  const data = columnsOrder?.length
```

Comment:
In my opinion we'll need to optimize this line to do not perform sort action for each table re-render as for large tables this could affect performance.